### DISCUSSION TITLE
所见即所得的笔记方式是趋势，大家怎么看呢

### DISCUSSION THREAD

**Discussion by world520ai** [tw93/MiaoYan#249] [Ideas]
像notion,语雀等都是所见即所得的笔记方式， 记笔记的时候和markdown语法混排，思路就没了，不知道大家同意这样的看法吗

**Comment by bz404** [comment]
认同.
实际上,对markdown方式的使用更多是程序员朋友们热衷一些.

fslkjf324234ewrwp ***@***.***>于2023年2月23日 周四上午10:00写道：

**Comment by peileiscott** [comment]
作者好像很早就在开发[即时编辑](https://gw.alipayobjects.com/os/k/hi/miaoyan.mp4)的功能了

**Comment by bz404** [comment]
peileiscott ***@***.***>于2023年3月6日 周一下午4:54写道：

> 作者好像很早就在开发即时编辑 <https://gw.alipayobjects.com/os/k/hi/miaoyan.mp4>的功能了
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tw93/MiaoYan/issues/220#issuecomment-1455726336>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIVWQPS4DQUOOEE3TNLF6JLW2WQ2TANCNFSM6AAAAAAVFBNREM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
期待👋

**Reply by shenweiyan** [reply]
这种混合起来的写法对不太懂 markdown 的新手挺友好的，但是也很容易导致 markdown 格式混乱，尤其是在导出和转移的时候会引发很多问题，熟悉了 markdown 后有时候还挺反感这种所见即所得的写法的~
